src/Serilog.Sinks.EventLog/Serilog.Sinks.EventLog.csproj

```diff
@@ -26,11 +26,6 @@
 
   <ItemGroup>
     <PackageReference Include="Serilog" Version="2.3.0" />
+    <PackageReference Include="System.Diagnostics.EventLog" Version="4.5.0-preview1-25914-04" />
```

Comment:
If we keep the .NET 4.5 target, this can then be placed in a `Condition=` block (like the deleted one a few lines down) so that on .NET Framework we'll continue to use the built-in assembly, only referencing this package under the .NET Standard targets.